tests/operator/operator.go

```diff
@@ -949,8 +950,10 @@ spec:
 			Expect(kv.GetGeneration()).To(Equal(generation))
 
 			By("Deleting patch from KubeVirt object")
-			kv.Spec.CustomizeComponents = v1.CustomizeComponents{}
-			kv, err = virtClient.KubeVirt(originalKv.Namespace).Update(context.Background(), kv, metav1.UpdateOptions{})
+
+			patchPayload, err = patch.New(patch.WithReplace("/spec/customizeComponents", v1.CustomizeComponents{})).GeneratePayload()
```

Comment:
@dsionov491 Are you sure it should be `v1.CustomizeComponents{}` and not `[]v1.CustomizeComponentsPatch{}`?

Reply:
hi, @enp0s3 , the CustomizeComponents is a struct that holds 2 fields 
1. Patches []CustomizeComponentsPatch
2. Flags *Flags

since i am replacing the parent struct it's should be v1.CustomizeComponents{}

unless you believe it better to replace just the patches list, i don't see any difference